core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -6028,3 +6059,4 @@ user_secrets_target_clusters:
 - hosted-mgmt
 - vsphere02
 vault_dptp_prefix: dptp
+
```

Comment:
Remove this line

Reply:
Removed now, I'm confulsed, the checking asked for a line earlier...

Reply:
@rrasouli It looks good now